app/models/item.rb

```diff
@@ -1,2 +1,8 @@
 class Item < ApplicationRecord
+  belongs_to :merchant
```

Comment:
Good catch with building the relationships. I might have to go do all of them in my current branch because I am testing practically all of them :/